scripts/rabbitmq-defaults

```diff
@@ -34,15 +34,15 @@ fi
 
 BOOT_MODULE="rabbit"
 
-CONFIG_FILE=${SYS_PREFIX}/etc/rabbitmq/rabbitmq
-LOG_BASE=${SYS_PREFIX}/var/log/rabbitmq
-MNESIA_BASE=${SYS_PREFIX}/var/lib/rabbitmq/mnesia
-ENABLED_PLUGINS_FILE=${SYS_PREFIX}/etc/rabbitmq/enabled_plugins
-GENERATED_CONFIG_DIR=${SYS_PREFIX}/var/lib/rabbitmq/config
-ADVANCED_CONFIG_FILE=${SYS_PREFIX}/etc/rabbitmq/advanced.config
-SCHEMA_DIR=${SYS_PREFIX}/var/lib/rabbitmq/schema
+CONFIG_FILE="$SYS_PREFIX"/etc/rabbitmq/rabbitmq
```

Comment:
Why delete ${brackets}?
Is that for consistency?
It could be just me, but it feels safer to have them.

Reply:
Yep consistency! `/` isn't a valid character to use in a variable name so it's all good here.